src/menu/MenuSongSelection.as

```diff
@@ -789,6 +786,10 @@ package menu
                 pane.scrollTo(scrollVal);
                 scrollbar.scrollTo(scrollVal);
             }
+
+            // Update Multiplayer Selection
+            if (mpUpdate && options.activeSongID != -1)
```

Comment:
For a valid song, is it possible that its song ID is -1?

Reply:
Only on Alt Engines if they intentionally set a valid song to have a level id of -1. But it's better to be safe when dealing with multiplayer logic just in the off-chance something odd happens.